packages/passport-auth/lib/Passport.js

```diff
@@ -76,7 +81,7 @@ class PassportAuthStrategy {
         }
         resolve({
           id: profile.id,
-          username: profile.username || null,
+          username: profile.displayName || null,
```

Comment:
Should this be:

```
username: profile.username || profile.displayName || null
```
?

I think different services return the profile with different sets of data.

But really, is the `displayName` a _`username`_? They're different things I think 🤔

Reply:
You're right, Facebook returns a `name` object. Twitter returns `username` and Google `displayName`

Reply:
I believe all of them gets normalized to username inside passportjs strategy